index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Meryl Streep</title>
+  <link rel="stylesheet" href="css.stylesheet" type="text/css" media="screen">
```

Comment:
The page needs to refer/link to the proper css file, `stylesheet.css` just like the resume page does.  🔨